### PR TITLE
Fixes duplicate check and general shitcode in human movement delay

### PR DESCRIPTION
**Pull request by 9600bauds**
Holy shitcode Batman. I saw this and I had to fix it.

Removes duplicate "if(istype(loc, /turf/space)) return -1" check.
Moves all the checks that can immediately return to the top, for obvious optimization
Moves all the checks that multiply the tally to the bottom (zero multiplied by two IS STILL ZERO)
Technically fixes slimepeople not being slowed down by Hyperzine I guess

### CODE REVIEW

**Comment by Shadowmech88** [comment]
> fixes slimepeople not being slowed down by Hyperzine

Slime people are supposed to be slowed down by hyperzine? Why?

**Comment by 9600bauds** [comment]
I don't know

File: code/modules/mob/living/carbon/human/human_movement.dm [hunk 1]
```diff
@@ -1,32 +1,31 @@
...
+			tally += (283.222 - bodytemperature) / 10 * 1.75
+	else if (undergoing_hypothermia())
+		tally += 2*undergoing_hypothermia()
 
-	if(reagents.has_reagent("hyperzine"))
-		if(dna.mutantrace == "slime")
-			tally *= 2
-		else
-			tally -= 10
+	//(VG EDIT disabling for now) handle_embedded_objects() //Moving with objects stuck in you can cause bad times.
```
Comment (ghost) [inline code/modules/mob/living/carbon/human/human_movement.dm]: change VG to /vg/ please

Comment (9600bauds) [inline code/modules/mob/living/carbon/human/human_movement.dm]: Later

**Comment by clusterfack** [comment]
I really really hate this proc you know

**Comment by 9600bauds** [comment]
I know. I don't blame you. I barely even know it and I already do too.

**Comment by ghost** [comment]
:+1:

**Pull request merged**